tutorials/proto_rename/renaming-proto.md

```diff
@@ -70,32 +70,29 @@ was changed to `string full_name`.
 `setFavoritePizzaTopping(Person.FavoritePizzaTopping.OLIVES_AND_PINEAPPLE)` 
 setter to 
 `setFavoritePizzaTopping(Person.FavoritePizzaTopping.SEAFOOD)` 
-in `writePerson()` method.
-- Run the command to read `person.protobin` file:
+in `writePersons()` method.
+- Run the command to read `persons.protobin` file:
 ```bash
 bazel run //tutorials/proto_rename:person_tool -- read
 ```
-You can see that we still read `person.protobin` file 
+You can see that we still read `persons.protobin` file 
 and `OLIVES_AND_PINEAPPLE` field was changed to `SEAFOOD`.
  
 ## Renaming message field
```

Comment:
I think it would be more interesting to rename the message itself - `Person`, to say, `Employee`, and see that it still works.

Reply:
Added renaming of a message to the tutorial.